### ISSUE TITLE
Dynamic placeholders causes a fatal error

### ISSUE THREAD

**Issue by ngryman** [sasstools/sass-lint#532]
Hi,

Dynamic placeholder names are reported as fatal errors when declared but not when they are extended.
I'm using `sass-lint 1.5.0`.

Code example:
```scss
// produces an error
%#{$placeholder-id} { color: red; }
// but the following is ok
'%#{$placeholder-id}' { color: red; }

// this does not produce an error
@extend %#{$placeholder-id};
```

**Comment by DanPurdy** [comment]
This is a problem with the version of gonzales-pe (our AST) we were using. It didn't have great support for interpolation.

I can confirm that all your above examples work in the latest version of gonzales-pe which we're currently  in the process of upgrading too. It should be complete and released shortly. 

See #495

**Comment by ngryman** [comment]
@DanPurdy Awesome, thanks for the quick reply!

**Comment by IlanFrumer** [comment]
+1

**Comment by bgriffith** [comment]
@ngryman Fixed in 1.6.
